### DISCUSSION TITLE
Suddenly getting menu data undefined when menu api called through swell js package

### DISCUSSION THREAD

**Discussion by renuka2605** [swellstores/community#210] [Help]
I am using swell js package to get data from swell. Suddenly menu api stopped working. There is no option for navigation menu. I have also tried below discussion link:
https://github.com/orgs/swellstores/discussions/200#discussioncomment-5551766  (Via Swell Admin Dashboard)

But I have not seen navigation menu link in it. Please guide me on this.

**Comment by Krislunde** [comment]
I am a dev at an agency trying to start using Swell for clients, so have been trying to onboarding myself with the platform today. I don't know what happened 4-5 days ago, but I'm finding issues everywhere:

Related to this issue:
- The Origin storefront when cloned from repo just bugs out and shows an error page because it can't request pages or navigation from my swell store. Pages or Navigation is nowhere to be found in the dashboard on a newly created account. Product page works fine though.

Other issues:
- The documentation has links to supposed pages with information about things like navigation for the Origin storefront that just 404's
- Quickstart links on the home page of the Swell dashboard, "Request demo", "Activate a storefront theme", and "Read the docs" just plain don't work/do anything. (Actually, "activate storefront theme" goes to 404 page.)
-  On one account, trying to create a new product leads to blank page with errors in console ("Cannot read properties of undefined (reading 'cost_tracking')"). Creating a new account fixed the issue -> but old account still broken with same issue. Seems like borked deploy or something.
-  i18n translation errors everywhere so nothing is labelled properly in the Origin storefront, and there seems to be css issues.

Coupled with all the other posts on here from whatever happened 4 days ago where it seems like peoples stores just got nuked, it's making me very uneasy. Has there been any official communication around whatever is going on?

**Reply by logeshswell** [reply]
@renuka2605 @Krislunde Please help me with your store-id. I will look into it

**Reply by Krislunde** [reply]
@logeshswell "superodd.swell.store" was the account with the product creation issue. It seems to have been fixed since yesterday (but I haven't heard from your support yet). 
"beans-beans.swell.store" is the current, new, account I'm using for testing / onboarding purposes.

All other aforementioned issues are still present. Plus: While looking for pages or navigation in the swell dashboard I found that Pages and Navigation are both referenced under content Models under Developer. When trying to add a content field to the Pages model (I was thinking maybe that would make it appear or something) the application just throws an error when trying to save: 

```
Error updating content model

**Field error: Root field 'content' is not a valid object path**

`{
  "field": "content.test_input"
}`
```

**Reply by logeshswell** [reply]
There is a gap in swell-cli which causes issue in setting up origin and horizon storefronts. Our engineers are almost very close to fix and roll out to production. I will keep you posted once it is released.
